The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/7848090504/job/21418438039) [(plain)](https://github.com/rust-lang/rust/commit/e59d9b171ed276ede55765c63c75381b47549e07/checks/21418438039/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
Prepare all required actions
Getting action download info
Download action repository 'actions/checkout@v4' (SHA:b4ffde65f46336ab88eb53be808477a3936bae11)
Download action repository 'actions/upload-artifact@v3' (SHA:a8a3f3ad30e3422c9c7b888a15615d19a852ae32)
Complete job name: PR - mingw-check-tidy
git config --global core.autocrlf false
shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
---
GITHUB_ACTION=__run_7
GITHUB_ACTIONS=true
GITHUB_ACTION_REF=
GITHUB_ACTION_REPOSITORY=
GITHUB_ACTOR=blyxyas
GITHUB_API_URL=https://api.github.com
GITHUB_BASE_REF=master
GITHUB_ENV=/home/runner/work/_temp/_runner_file_commands/set_env_029e9635-d90a-4b4e-b857-3fbe1442f1ae
GITHUB_EVENT_NAME=pull_request
---
GITHUB_SERVER_URL=https://github.com
GITHUB_SHA=3df775e3bfc4fd7235cbd5674de1d7f36a79ee16
GITHUB_STATE=/home/runner/work/_temp/_runner_file_commands/save_state_029e9635-d90a-4b4e-b857-3fbe1442f1ae
GITHUB_STEP_SUMMARY=/home/runner/work/_temp/_runner_file_commands/step_summary_029e9635-d90a-4b4e-b857-3fbe1442f1ae
GITHUB_TRIGGERING_ACTOR=blyxyas
GITHUB_WORKFLOW_REF=rust-lang/rust/.github/workflows/ci.yml@refs/pull/120853/merge
GITHUB_WORKFLOW_SHA=3df775e3bfc4fd7235cbd5674de1d7f36a79ee16
GITHUB_WORKSPACE=/home/runner/work/rust/rust
GOROOT_1_19_X64=/opt/hostedtoolcache/go/1.19.13/x64
---
COPY scripts/sccache.sh /scripts/
RUN sh /scripts/sccache.sh

COPY host-x86_64/mingw-check/reuse-requirements.txt /tmp/
RUN pip3 install --no-deps --no-cache-dir --require-hashes -r /tmp/reuse-requirements.txt \
    && pip3 install virtualenv
COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/

# NOTE: intentionally uses python2 for x.py so we can test it still works.
# NOTE: intentionally uses python2 for x.py so we can test it still works.
# validate-toolstate only runs in our CI, so it's ok for it to only support python3.
ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test \
           --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile --allow-unsafe --generate-hashes reuse-requirements.in
---

#10 [5/8] COPY host-x86_64/mingw-check/reuse-requirements.txt /tmp/
#10 DONE 0.0s

#11 [6/8] RUN pip3 install --no-deps --no-cache-dir --require-hashes -r /tmp/reuse-requirements.txt     && pip3 install virtualenv
#11 0.701   Downloading binaryornot-0.4.4-py2.py3-none-any.whl (9.0 kB)
#11 0.717 Collecting boolean-py==4.0
#11 0.724   Downloading boolean.py-4.0-py3-none-any.whl (25 kB)
#11 0.740 Collecting chardet==5.1.0
---
#11 3.825 Building wheels for collected packages: reuse
#11 3.826   Building wheel for reuse (pyproject.toml): started
#11 4.155   Building wheel for reuse (pyproject.toml): finished with status 'done'
#11 4.157   Created wheel for reuse: filename=reuse-1.1.0-cp310-cp310-manylinux_2_35_x86_64.whl size=181117 sha256=f5f58750481f69515c2c0d1d503daf565e2565c370d07fc6aeb95fe3498b4269
#11 4.157   Stored in directory: /tmp/pip-ephem-wheel-cache-sk1yy6dq/wheels/c2/3c/b9/1120c2ab4bd82694f7e6f0537dc5b9a085c13e2c69a8d0c76d
#11 4.159 Installing collected packages: boolean-py, binaryornot, setuptools, reuse, python-debian, markupsafe, license-expression, jinja2, chardet
#11 4.182   Attempting uninstall: setuptools
#11 4.182     Found existing installation: setuptools 59.6.0
#11 4.183     Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr
#11 4.183     Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr
#11 4.184     Can't uninstall 'setuptools'. No files were found to uninstall.
#11 4.855 Successfully installed binaryornot-0.4.4 boolean-py-4.0 chardet-5.1.0 jinja2-3.1.2 license-expression-30.0.0 markupsafe-2.1.1 python-debian-0.1.49 reuse-1.1.0 setuptools-66.0.0
#11 4.855 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#11 5.376 Collecting virtualenv
#11 5.427   Downloading virtualenv-20.25.0-py3-none-any.whl (3.8 MB)
#11 5.555      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.8/3.8 MB 30.1 MB/s eta 0:00:00
#11 5.606 Collecting platformdirs<5,>=3.9.1
#11 5.613   Downloading platformdirs-4.2.0-py3-none-any.whl (17 kB)
#11 5.647 Collecting filelock<4,>=3.12.2
#11 5.653   Downloading filelock-3.13.1-py3-none-any.whl (11 kB)
#11 5.673 Collecting distlib<1,>=0.3.7
#11 5.680   Downloading distlib-0.3.8-py2.py3-none-any.whl (468 kB)
#11 5.691      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 51.3 MB/s eta 0:00:00
#11 5.773 Installing collected packages: distlib, platformdirs, filelock, virtualenv
#11 5.935 Successfully installed distlib-0.3.8 filelock-3.13.1 platformdirs-4.2.0 virtualenv-20.25.0
#11 DONE 6.0s

#12 [7/8] COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
#12 DONE 0.0s
---
DirectMap4k:      192448 kB
DirectMap2M:     7147520 kB
DirectMap1G:    11534336 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
    Finished dev [unoptimized] target(s) in 0.03s
##[endgroup]
downloading https://ci-artifacts.rust-lang.org/rustc-builds-alt/e28fae52d99d7c14bf0890d1f2b13c2c34fa8932/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz
extracting /checkout/obj/build/cache/llvm-e28fae52d99d7c14bf0890d1f2b13c2c34fa8932-true/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz to /checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm
---
   Compiling tidy v0.1.0 (/checkout/src/tools/tidy)
    Finished release [optimized] target(s) in 25.05s
##[endgroup]
fmt check
##[error]Diff in /checkout/compiler/rustc_lint/src/late.rs at line 368:
     if late_module_passes.is_empty() {
         late_lint_mod_inner(tcx, module_def_id, context, builtin_lints);
     } else {
-        let mut passes: Vec<_> = late_module_passes
-            .iter()
-            .map(|mk_pass| (mk_pass)(tcx))
-            .collect();
+        let mut passes: Vec<_> = late_module_passes.iter().map(|mk_pass| (mk_pass)(tcx)).collect();
         passes.push(Box::new(builtin_lints));
         let pass = RuntimeCombinedLateLintPass { passes: &mut passes[..] };
         late_lint_mod_inner(tcx, module_def_id, context, pass);
Running `"/checkout/obj/build/x86_64-unknown-linux-gnu/rustfmt/bin/rustfmt" "--config-path" "/checkout" "--edition" "2021" "--unstable-features" "--skip-children" "--check" "/checkout/compiler/rustc_lint/src/async_fn_in_trait.rs" "/checkout/compiler/rustc_lint/src/types.rs" "/checkout/compiler/rustc_lint/src/drop_forget_useless.rs" "/checkout/library/alloc/src/collections/linked_list/tests.rs" "/checkout/compiler/rustc_lint/src/late.rs" "/checkout/compiler/rustc_lint/src/builtin.rs" "/checkout/compiler/rustc_lint/src/context.rs" "/checkout/compiler/rustc_lint/src/let_underscore.rs"` failed.
If you're running `tidy`, try again with `--bless`. Or, if you just want to format code, run `./x.py fmt` instead.
  local time: Fri Feb  9 18:28:34 UTC 2024
  network time: Fri, 09 Feb 2024 18:28:34 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/compiler/rustc_lint/src/late.rs
+++ b/compiler/rustc_lint/src/late.rs
@@ -368,10 +368,7 @@ pub fn late_lint_mod<'tcx, T: LateLintPass<'tcx> + 'tcx>(
     if late_module_passes.is_empty() {
         late_lint_mod_inner(tcx, module_def_id, context, builtin_lints);
     } else {
-        let mut passes: Vec<_> = late_module_passes
-            .iter()
-            .map(|mk_pass| (mk_pass)(tcx))
-            .collect();
+        let mut passes: Vec<_> = late_module_passes.iter().map(|mk_pass| (mk_pass)(tcx)).collect();
         passes.push(Box::new(builtin_lints));
         let pass = RuntimeCombinedLateLintPass { passes: &mut passes[..] };
         late_lint_mod_inner(tcx, module_def_id, context, pass);

